machinelearning/llama.cpp.md

```diff
@@ -0,0 +1,73 @@
+# Large Language Model (LLM) inference on Graviton CPUs with llama.cpp
+
+**Introduction**
+
+The main goal of [llama.cpp](https://github.com/ggerganov/llama.cpp) is to enable LLM inference with minimal setup and state-of-the-art performance on a wide variety of hardware. It's a plain C/C++ implementation without any dependencies. It supports quantized general matrix multiply and add (GEMM) kernels for faster inference and reduced memory use. The quantized GEMM kernels are optimized for AWS Graviton processors using Arm Neon and SVE based matrix multiply and add (MMLA) instructions. This document covers how to build and run llama.cpp efficiently for LLM inference on AWS Graviton based Amazon EC2 instances.
```

Comment:
minor: "multiply and add kernels" may sound like two different kernels to the unseasoned reader. To reinforce the idea that is a single kernel, how about "multiply-add kernels" or "multiply-and-add kernels"?